elrond-wasm-debug/src/api/managed_types/big_int_api_mock.rs

```diff
@@ -185,7 +186,7 @@ impl BigIntApi for DebugApi {
     fn bi_pow(&self, dest: Self::BigIntHandle, x: Self::BigIntHandle, y: Self::BigIntHandle) {
         let bi_x = self.bi_get(x);
         let bi_y = self.bi_get(y);
-        let exp = big_int_to_i64(&bi_y).unwrap() as usize;
+        let exp = big_int_to_i64(&bi_y).unwrap().try_into().unwrap();
```

Comment:
Unwrap `panic`s. We need to `signal_error`.

`let exp = big_int_to_i64(&bi_y).unwrap().try_into().unwrap_or_else(...)`